assets/js/analytics-events.tsx

```diff
@@ -192,23 +193,38 @@ function SidebarNavItem({ name }: { name: string }) {
   );
 }
 
-function ErrorPage({ error, url }: { error: Error; url: string }) {
+function AlertComponent({
```

Comment:
Maybe drop the "Component" suffix, since it's assumed.

```suggestion
function Alert({
```